README.md

```diff
@@ -80,10 +109,10 @@ In case no `id` is defined, LoopBack adds ***id*** with Cassandra connector's de
 LoopBack notation:
 
 ```javascript
-    cass = db.define('cass', {
-      str: String,
-      num: Number,
-      });
+cass = db.define('cass', {
+  str: String,
+  num: Number,
+  });
```

Comment:
I intentionally made them `dry` and pure technical because they are simple enough to read through and because adding real-life flavor does not help much (since it's simple anyway).

The sorting example is different; it helps people intuitively get how the rows should be sorted.  Then, read on and get an idea how it's implemented to get the result.

And the examples are used in the C* specific test code.

But, i respect your commend @raymondfeng.  If you still think spending a few hours and making it more leal-life, i will do that.  What do you think?